### ISSUE TITLE
Allow import of raw content 

### ISSUE THREAD

**Issue by KDean-Dolphin** [microsoft/advanced-formula-environment#59]
Gist is useful, but it has a long-standing issue in that it doesn't support organizations. Anything published to a Gist has to be published under an individual user's account, which we don't want to do.

Please add support for importing raw content, e.g., https://raw.githubusercontent.com/microsoft/advanced-formula-environment/main/examples/Lib.md

**Comment by jack-williams** [comment]
By raw content do you mean from a local file?

**Comment by KDean-Dolphin** [comment]
No, I mean from HTTP endpoint as per example above. If my GitHub repository is public, I could share a link like the one above and users could be assured of getting the latest and greatest there.

**Comment by derfasthirnlosenick** [comment]
So similar to #53 ?

**Comment by KDean-Dolphin** [comment]
Similar, yes. I would prefer to see #53 fully implemented, but in the meantime adding a straightforward HTTP GET to a text endpoint strikes me as a simpler ask and it gets around the problem of the lack of organization support for Gists.
